spec/Subgraph.coffee

```diff
@@ -209,6 +209,7 @@ describe 'Graph component', ->
     gr.addOutport 'out', 'Split', 'out'
     it 'should emit a ready event after network has been loaded', (done) ->
       @timeout 6000
+      c.baseDir = root
```

Comment:
Because some additional error handling was added to the Subgraph component. Previously these had failed, but silently